### PR TITLE
[zh] Sych web page for topo manager

### PR DESCRIPTION
**Pull request by chenxuc**
This is part of work in umbrella issue:
[zh] Umbrella issue: pages out of sync in tasks section #26178

Topology Manageer (L):
 content/zh/docs/tasks/administer-cluster/topology-manager.md

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/lgtm

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 1]
```diff
@@ -120,6 +132,115 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
...
+* 所有容器可以被分配到一个单一的 NUMA 节点；
+* 所有容器可以被分配到一个共享的 NUMA 节点集。
+
+<!--
+The total amount of particular resource demanded for the entire pod is calculated according to [effective requests/limits](/docs/concepts/workloads/pods/init-containers/#resources) formula, and thus, this total value is equal to the maximum of:
+* the sum of all app container requests,
+* the maximum of init container requests,
+for a resource.
+-->
+整个 Pod 所请求的某种资源总量是根据 [有效 request/limit](/docs/concepts/workloads/pods/init-containers/#resources) 公式来计算的，
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:206]: 注意折行和链接。

```suggestion
整个 Pod 所请求的某种资源总量是根据
[有效 request/limit](/zh/docs/concepts/workloads/pods/init-containers/#resources)
公式来计算的，
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 2]
```diff
@@ -120,6 +132,115 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
...
+在 `single-numa-node` 策略下，只有当可能的分配方案中存在合适的 NUMA 节点集时，Pod 才会被接受。
+重新考虑上述的例子：
+
+<!--
+* a set containing only a single NUMA node - it leads to pod being admitted,
+* whereas a set containing more NUMA nodes - it results in pod rejection (because instead of one NUMA node, two or more NUMA nodes are required to satisfy the allocation).
+-->
+* 节点集只包含单个 NUMA 节点时，Pod 就会被接受，
+* 然而，节点集包含多个 NUMA 节点时，
+Pod 就会被拒绝（因为满足该分配方案需要两个或以上的 NUMA 节点，而不是单个 NUMA 节点）。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:231]:
```suggestion
* 然而，节点集包含多个 NUMA 节点时，Pod 就会被拒绝
  （因为满足该分配方案需要两个或以上的 NUMA 节点，而不是单个 NUMA 节点）。
```

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 3]
```diff
@@ -120,6 +132,115 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
...
+
+<!--
+To recap, Topology Manager first computes a set of NUMA nodes and then tests it against Topology Manager policy, which either leads to the rejection or admission of the pod.
+-->
+简要地说，拓扑管理器首先计算出 NUMA 节点集，然后用拓扑管理器策略来检验它，
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:236]:
```suggestion
简要地说，拓扑管理器首先计算出 NUMA 节点集，然后使用拓扑管理器策略来测试该集合，
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the review. I've updated the commit.

File: content/zh/docs/tasks/administer-cluster/topology-manager.md [hunk 4]
```diff
@@ -120,6 +132,117 @@ To align CPU resources with other requested resources in a Pod Spec, the CPU Man
...
+* 所有容器可以被分配到一个共享的 NUMA 节点集。
+
+<!--
+The total amount of particular resource demanded for the entire pod is calculated according to [effective requests/limits](/docs/concepts/workloads/pods/init-containers/#resources) formula, and thus, this total value is equal to the maximum of:
+* the sum of all app container requests,
+* the maximum of init container requests,
+for a resource.
+-->
+整个 Pod 所请求的某种资源总量是根据
+[有效 request/limit](/docs/concepts/workloads/pods/init-containers/#resources)
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/topology-manager.md:207]:
```suggestion
[有效 request/limit](/zh/docs/concepts/workloads/pods/init-containers/#resources)
```

**Comment by chenxuc** [comment]
@tengqm I've fixed the link. thanks.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**